src/amo/components/AddonReviewCard/styles.scss

```diff
@@ -123,6 +123,14 @@
 
     @include margin-start(12px);
   }
+
+  .ShowMoreCard-contents::after {
+    background: linear-gradient(rgba(255, 255, 255, 0), #f4f9ff);
```

Comment:
It's noted in the ShowMoreCard stylesheet that for the linear-gradient `rgba(255, 255, 255, 0)` is used instead of `transparent` to prevent a gray line in Safari.

The variable with the lightest shade of blue is $blue-50.  Can I create a variable that is set to `#f4f9ff`?